vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/SampleOverviewComponent.java

```diff
@@ -352,6 +371,48 @@ private void setupSearchFieldForExperimentTabs(String experimentName,
         }
       });
     }
+
+
+    private void processBatchRegistration(BatchRegistrationContent batchRegistrationContent) {
+      //Todo add Batch name here and trigger processSampleCreation() method
+      Result<BatchId, BatchRegistrationService.ResponseCode> batch = batchRegistrationService.registerBatch(
+          batchRegistrationContent.batchLabel(), batchRegistrationContent.isPilot());
+      batch.onValue(result -> {
+        batchRegistrationDialog.resetAndClose();
+        lastCreatedBatch = result;
+        //ToDo Replace Values
+      }).onError(e -> {
+        //ToDo What should happen here?
+      });
+    }
+
+    private void processSampleRegistration(
+        List<SampleRegistrationContent> sampleRegistrationContentList) {
+      //ToDo Sample Comments are currently not stored
+      sampleRegistrationContentList.forEach(sampleRegistrationContent -> {
+        //ToDo Where should these domain objects be generated?
```

Comment:
Please address or create an Jira issue

Reply:
As discussed i removed the toDos since it's unclear how the comments will be used